rag-sql-router/app.py

```
@@ -0,0 +1,454 @@
+                        # If it's a dictionary string (like the context not found case), extract the response
+                        if response_content.startswith("{'response':") or response_content.startswith('{"response":'):
+                            try:
+                                import ast
+                                # Try to safely evaluate the string as a dictionary
+                                response_dict = ast.literal_eval(response_content)
+                                if isinstance(response_dict, dict) and 'response' in response_dict:
+                                    response_content = response_dict['response']
+                                    # Update trust_score if it's in the dict
+                                    if 'trust_score' in response_dict and trust_score is None:
+                                        trust_score = response_dict['trust_score']
+                            except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)